lib/modules/model/ModelsController.ts

```diff
@@ -64,6 +65,10 @@ export class ModelsController {
           handler: this.listMeasures.bind(this),
           http: [{ path: "device-manager/models/measures", verb: "get" }],
         },
+        updateAsset: {
+          handler: this.updateAsset.bind(this),
+          http: [{ path: "device-manager/models/assets/:model", verb: "put" }],
```

Comment:
I think it is more a 'patch' than a 'put' because it does not allow to replace the entire asset with the provided playload (case on measures condition in the model service)